### PR TITLE
[release] Send: speed up or cancel transaction

### PR DESCRIPTION
**Pull request by MarnixCroes**
first merge https://github.com/zkSNACKs/WasabiDoc/pull/1566/ as it contains the image file

### CODE REVIEW

**Comment by MarnixCroes** [comment]
the only things that's missing imo is some info about how much it will speed it up

File: docs/using-wasabi/Send.md [hunk 1]
```diff
@@ -158,3 +158,29 @@ This first node will gossip the transaction throughout the network, then miners
...
+The miners are incentivised to mine the transaction with the higher fee, as this will earn them more money.
+If the new higher fee rate paying transaction is confirmed, the old transaction can be considered "replaced".
+
+Since Wasabi version [2.0.4](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4) it is possible to easily speed up or cancel a pending transaction.
+Speeding up and cancelling are similar to each other, the main difference being that a _Cancel Transaction_ will be send to a new address which is owned by the user himself (this is all done automatically).
+The transaction is then "cancelled" because the bitcoin (minus additional fees) is returned to the user's wallet.
+
+Speeding up or cancelling a transaction costs additional fees, because a new bitcoin transaction with a higher fee (than the previous transaction) has to be sent.
+
+Wasabi first tries to utilize [RBF](glossary/Glossary-GeneralBitcoin.html#replace-by-fee-rbf), but if that's not possible it will do [CPFP](glossary/Glossary-GeneralBitcoin.html#child-pays-for-parent-cpfp).
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:176]:
```suggestion
Wasabi first tries to utilize [RBF](glossary/Glossary-GeneralBitcoin.html#replace-by-fee-rbf), if that's not possible it tries to do [CPFP](glossary/Glossary-GeneralBitcoin.html#child-pays-for-parent-cpfp).
```

File: docs/using-wasabi/Send.md [hunk 2]
```diff
@@ -158,3 +158,29 @@ This first node will gossip the transaction throughout the network, then miners
...
+
+To Speed Up or Cancel a Transaction, right-click on the pending transaction in the history.
+
+![Speed Up History](/SpeedUpHistory.png "Speed Up History")
+
+Select it and then confirm that you want to pay the additional fee for this action.
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:182]:
```suggestion
Click one of the two options and then confirm that you want to pay the additional fee for this action.
```

File: docs/using-wasabi/Send.md [hunk 3]
```diff
@@ -158,3 +158,29 @@ This first node will gossip the transaction throughout the network, then miners
...
+After which the succesfully speed up/cancelled dialog is displayed.
+
+In the history, the transaction will now have a rocket icon (speeded up) or a cross (cancelled).
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:185]:
```suggestion
In the history, the transaction will now have a rocket (speeded up) or a cross (cancelled) icon.
```

File: docs/using-wasabi/Send.md [hunk 4]
```diff
@@ -158,3 +158,29 @@ This first node will gossip the transaction throughout the network, then miners
...
+The transaction is still pending, but it will be confirmed sooner than previously.
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:186]:
```suggestion
The transaction is still pending, but it should be confirmed sooner than the initial transaction.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
Since we have an FAQ on how to speed up a transaction, should we add a separate FAQ for how to cancel a transaction?
(This question is not related to this PR)

**Pull request merged**